CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 - Added support for `resample` frequency `W`, `ME`, `YE` with `closed = "left"`.
 - Added support for `DataFrame.rolling.corr` and `Series.rolling.corr` for `pairwise = False` and int `window`.
 - Added support for string time-based `window` and `min_periods = None` for `Rolling`.
+- Added support for `pd.read_sas` (Uses local pandas for processing).
```

Comment:
should we either always use `local pandas` or always use `native pandas`?  I see below it has `native pandas`

Reply:
Good point. Fixed.